app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -464,4 +464,217 @@
             }
         }
     }
+
+    /* SECTION__NEWSLETTER */
+
+    .section__newsletter {
+        padding: 0;
+        background-color: @color-maksym__main-light-gray;
+        .newsletter__wraper {
+            div {
+                width: 50%;
+                p {
+                    width: 100%;
+                }
+                .content,
+                form {
+                    width: 100%;
+                }
+            }
+            .newsletter__sub-title {
+                width: 100%;
+            }
+            .lib-vendor-prefix-display(flex);
+            justify-content: space-between;
+            align-items: center;
+            margin-left: auto;
+            margin-right: auto;
+            max-width: 1280px;
+            width: 100%;
+        }
+        .block.newsletter {
+            width: 50%;
+            margin: 0;
+            input {
+                display: inline-block;
+                width: 100%;
+                background: transparent;
+                border: 1px solid @color-maksym__main-gray;
+                height: 49px;
+                line-height: 49px;
+                color: @color-maksym__main-gray;
+                .lib-font-size(14);
+                padding-left: 20px;
+                padding: 0 0 0 20px;
+                margin: 0;
+            }
+            button {
+                padding: 17px 35px;
+                border: 0;
+                border-radius: 0;
+                margin: 0;
+                color: @color-maksym__white;
+                background: @color-maksym__main-black;
+                font-style: normal;
+                .lib-font-size(14);
+                font-weight: @font-weight__regular;
+                text-transform: uppercase;
+                letter-spacing: 1px;
+            }
+            .block.newsletter .action.subscribe {
+                border-radius: 0;
+            }
+        }
+        .block.newsletter {
+            .field {
+                .control:before {
+                    display: none;
+                }
+            }
+        }
+        .block.newsletter {
+            .form.subscribe {
+                display: flex;
+                align-items: center;
+                width: 100%;
+            }
+        }
+        .block.newsletter {
+            .field.newsletter {
+                max-width: 46%;
+            }
+        }
+    }
+
+    /* FOOTER */
+    div.footer.content {
+        max-width: 100%;
+        margin: 0;
+        border: 0;
+    }
+    .footer {
+        h3 {
+            margin-bottom: 40px;
+            .lib-font-size(16);
+            font-weight: @font-weight__heavier;
+            line-height: 16px;
+            text-transform: uppercase;
+        }
+
+        .footer__links-thumb__sub-list__item {
+            display: flex;
+            align-items: center;
```

Comment:
Please use mixin for `display: flex `and `align-items: center`

Reply:
done